cli.md

```diff
@@ -21,7 +21,7 @@ The Fabric command line tools (CLI) can be used by mod developers to generate ne
 The Fabric CLI tools require the [deno runtime](https://deno.com/runtime) to be installed on your system. Deno is a cross platform JavaScript runtime that allows us to use one code base between the web and desktop. After following the [deno installation](https://deno.com/manual/getting_started/installation) instructions, run the following command to globally install the fabric CLI tools:
 
 <code class="command">
-deno install -A -n fabric https://fabricmc.net/cli
+deno install --unstable -A -n fabric https://fabricmc.net/cli
```

Comment:
No, we shouldnt require people to use unstable features, this will also break anyone upgrading an existing installs of the cli.

Reply:
Added a separate install command for unstable.